google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/dom/AppEngineStandardDomElement.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2016 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine.dom;
+
+import com.intellij.util.xml.DomElement;
+import com.intellij.util.xml.GenericDomValue;
+
+/**
+ * DOM representation of the App Engine standard environment appengine-web.xml file.
+ */
+public interface AppEngineStandardDomElement extends DomElement {
+
+  GenericDomValue<String> getVm();
```

Comment:
I think this can return Boolean. and then can be isVm().   We actually have the beginning of this in the Android Studio plugin uder com.google.gct.studio.appengine.dom